Jasperserver persistence nami

raquel-campuzano opened the pull request:
Using Wordpress as a model.

sameersbn commented:
@raquel-campuzano can you please rebase this PR to the master?

sameersbn commented on README.md:
```diff
@@ -81,29 +94,32 @@ services:
-      - /bitnami/jasperserver
-      - /bitnami/tomcat
+      - /path/to/jasperserver-persistence:/bitnami/jasperserver
+      - /path/to/tomcat-persistence:/bitnami/tomcat
```
I am not a 100% if a volume mount is required for tomcat. but in uncertainty i would default to mounting a volume. Could you please add this volume in the snippet at https://github.com/raquel-campuzano/bitnami-docker-jasperserver/#run-the-application-using-docker-compose and also the `docker-compose.yaml` file in the repo.

sameersbn commented on README.md:
```diff
@@ -81,29 +94,32 @@ services:
-      - /your/local/path/bitnami/jasperserver:/bitnami/jasperserver
-      - /your/local/path/bitnami/tomcat:/bitnami/tomcat
+      - /path/to/bitnami/jasperserver:/bitnami/jasperserver
+      - /path/to/bitnami/tomcat:/bitnami/tomcat
```
Looks like this `mounts` block here is a error. Actually this whole snippet is wrong. Could you rewrite the snippet based on the `docker-compose.yml` file. It's weird that no one noticed it.

raquel-campuzano commented:
The `docker-compose.yml` file had some wrong snippets in the "Persistence your application" section. I removed this snippets and rewritten it using the`docker-compose.yml` as reference.

sameersbn commented on README.md:
````diff
@@ -203,10 +223,13 @@ To backup your application data follow these steps:
+```bash
+$ docker cp /your/local/path/bitnami:/bitnami/jasperserver
+```
+=======
````
this block should go.

sameersbn commented on README.md:
````diff
@@ -112,10 +130,12 @@ Bitnami provides up-to-date versions of JasperReports, including security patche
-  ```
+```bash
+$ docker pull bitnami/jasperserver:latest
+```
````
above block should go
